mud-contracts/world/src/modules/smart-character/Utils.sol

```diff
@@ -7,7 +7,7 @@ import { WorldResourceIdLib } from "@latticexyz/world/src/WorldResourceId.sol";
 import { RESOURCE_SYSTEM } from "@latticexyz/world/src/worldResourceTypes.sol";
 import { ResourceIds } from "@latticexyz/store/src/codegen/tables/ResourceIds.sol";
 
-import { SMART_CHARACTER_SYSTEM_NAME } from "@eveworld/common-constants/src/constants.sol";
+import { SMART_CHARACTER_SYSTEM_NAME, FRONTIER_WORLD_DEPLOYMENT_NAMESPACE } from "@eveworld/common-constants/src/constants.sol";
```

Comment:
The ClassConfig table is something that gets deployed in the world not the SOF, to be available for Systems to use, I don't think it is really attached to the SOF directly.. it just stores classId values for triggering entity tagging in  the SOF. We could think about coupling this with the SOF later. Since ti does handle SOF entity value management.